mod-template-engine/src/main/resources/domain/mod-template-engine/features/templateRequest.feature

```diff
@@ -1,21 +1,77 @@
 Feature: Template processing requests tests
 
-  @Undefined
-  Scenario: Post templateProcessingRequest should return 422 if request did not pass validation
-    * print 'undefined'
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+    * def templateId = call uuid1
 
-  @Undefined
   Scenario: Post templateProcessingRequest should return 200 and templateProcessingResult
-    * print 'undefined'
+    * def requestEntity = read('samples/template-request-entity.json')
+
+    Given path 'templates'
+    When method GET
+    Then status 200
+
+    * requestEntity.templateId = response.templates[0].id
+    * requestEntity.outputFormat = response.templates[0].outputFormats[0]
+
+    Given path 'template-request'
+    And request requestEntity
+    When method POST
+    Then status 200
 
-  @Undefined
   Scenario: Post templateProcessingRequest should return 400 if template does not exist
-    * print 'undefined'
+    * def requestEntity = read('samples/template-request-entity.json')
+
+    Given path 'template-request'
+    And request requestEntity
+    When method POST
+    Then status 400
```

Comment:
please check the error message if it is possible

Reply:
done